jib-cli/src/main/java/com/google/cloud/tools/jib/cli/ArtifactProcessors.java

```diff
@@ -80,6 +87,38 @@ public static ArtifactProcessor from(
     }
   }
 
+  /**
+   * Creates a {@link ArtifactProcessor} instance.
+   *
+   * @param warPath path to the war
+   * @param cacheDirectories the location of the relevant caches
+   * @param warOptions war cli options
+   * @param commonContainerConfigCliOptions common cli options shared between jar and war command
+   * @return ArtifactProcessor
+   * @throws InvalidImageReferenceException if base image reference is invalid
+   */
+  public static ArtifactProcessor fromWar(
+      Path warPath,
+      CacheDirectories cacheDirectories,
+      War warOptions,
+      CommonContainerConfigCliOptions commonContainerConfigCliOptions)
+      throws InvalidImageReferenceException {
+    Optional<AbsoluteUnixPath> appRoot = warOptions.getAppRoot();
+    Optional<String> baseImage = commonContainerConfigCliOptions.getFrom();
+    boolean isCustomJetty =
+        baseImage.isPresent()
+            && ImageReference.parse(baseImage.get()).getRepository().contains("jetty");
```

Comment:
It should be `.equals()` rather than `.contains()`. And isn't it that `getRepository()` returns `library/jetty` for Docker Hub images? And perhaps check `getRegistry()` too just in case?